source/guide_guillotine.rst

```diff
@@ -0,0 +1,280 @@
+
+.. code-block:: ini
+
+  [program:guillotine]
+  environment=APP_ENV=production
+  directory=%(ENV_HOME)s/guillotine
+  command=/opt/uberspace/etc/%(ENV_USER)s/binpaths/ruby/rackup config.ru --host 0.0.0.0 --port 9292
+
+.. include:: includes/supervisord.rst
+
+If it's not in state RUNNING, check your configuration.
+
+Create short URL
+================
+
+Use your HTTP tool of choice (e.g. ``curl``) to make a POST request to create a short URL.
+
+.. code-block:: console
+  :emphasize-lines: 1-2
+
+  [isabell@stardust ~]$ curl https://isabell.uber.space \
+    --form "url=https://uberspace.de"
+  https://isabell.uber.space/D_deZA
+  [isabell@stardust ~]$
+
+Define your own short code like so:
+
+.. code-block:: console
+  :emphasize-lines: 1-3
+
+  [isabell@stardust ~]$ curl https://isabell.uber.space \
+    --form "url=https://lab.uberspace.de" \
+    --form "code=lab"
+  https://isabell.uber.space/lab
+  [isabell@stardust ~]$
+
+Updates
+=======
+
+.. note:: Check the update feed_ regularly to stay informed about the newest version.
+
+Use :manual_anchor:`gem <lang-ruby.html#gem>` to update dependencies like so:
+
+.. code-block:: console
+  :emphasize-lines: 1
+
+  [isabell@stardust ~]$ gem update
+  Updating installed gems
+  Updating addressable
+  Fetching addressable-2.6.0.gem
+  Fetching public_suffix-3.1.1.gem
+  WARNING:  You don't have /home/isabell/.gem/ruby/2.6.0/bin in your PATH,
+  	  gem executables will not run.
+  Successfully installed public_suffix-3.1.1
+  Successfully installed addressable-2.6.0
+  Updating bigdecimal
+  Fetching bigdecimal-1.4.4.gem
+  Building native extensions. This could take a while...
+  Successfully installed bigdecimal-1.4.4
+  Updating csv
+  Fetching csv-3.1.1.gem
+  Successfully installed csv-3.1.1
+  Updating fileutils
+  Fetching fileutils-1.2.0.gem
+  Successfully installed fileutils-1.2.0
+  Updating io-console
+  Fetching io-console-0.4.8.gem
+  Building native extensions. This could take a while...
+  Successfully installed io-console-0.4.8
+  Updating json
+  Fetching json-2.2.0.gem
+  Building native extensions. This could take a while...
+  Successfully installed json-2.2.0
+  Updating rack
+  Fetching rack-2.0.7.gem
+  Successfully installed rack-2.0.7
+  Updating rack-protection
+  Fetching rack-protection-2.0.5.gem
+  Successfully installed rack-protection-2.0.5
+  Updating rdoc
+  Fetching rdoc-6.1.1.gem
+  Successfully installed rdoc-6.1.1
+  Updating rexml
+  Fetching rexml-3.2.2.gem
+  Successfully installed rexml-3.2.2
+  Updating rss
+  Fetching rss-0.2.8.gem
+  Successfully installed rss-0.2.8
+  Updating sinatra
+  Fetching sinatra-2.0.5.gem
+  Fetching mustermann-1.0.3.gem
+  Successfully installed mustermann-1.0.3
+  Successfully installed sinatra-2.0.5
+  Gems updated: addressable public_suffix bigdecimal csv fileutils io-console json rack rack-protection rdoc rexml rss mustermann sinatra
+  [isabell@stardust ~]$
```

Comment:
Please add a note to restart the service and check its log here.

Reply:
Good point, added.